src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java

```diff
@@ -1,26 +1,7 @@
 package com.nekomaster1000.infernalexp.init;
 
 import com.nekomaster1000.infernalexp.InfernalExpansion;
-import com.nekomaster1000.infernalexp.blocks.BasalticMagmaBlock;
-import com.nekomaster1000.infernalexp.blocks.BuriedBoneBlock;
-import com.nekomaster1000.infernalexp.blocks.CrumblingBlackstoneBlock;
-import com.nekomaster1000.infernalexp.blocks.DullthornsBlock;
-import com.nekomaster1000.infernalexp.blocks.DullthornsBlockBlock;
-import com.nekomaster1000.infernalexp.blocks.FungusCapBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowCampfireBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowFireBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowSandBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowTorchBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowWallTorchBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowdustBlock;
-import com.nekomaster1000.infernalexp.blocks.LuminousFungusBlock;
-import com.nekomaster1000.infernalexp.blocks.NetherrackPathBlock;
-import com.nekomaster1000.infernalexp.blocks.NetherCarpetBlock;
-import com.nekomaster1000.infernalexp.blocks.ShroomlightFungusBlock;
-import com.nekomaster1000.infernalexp.blocks.SmoothGlowstonePressurePlateBlock;
-import com.nekomaster1000.infernalexp.blocks.SoulSoilPathBlock;
-import com.nekomaster1000.infernalexp.blocks.TrappedGlowSandBlock;
-import com.nekomaster1000.infernalexp.blocks.VerticalSlabBlock;
```

Comment:
revert this i stg

Reply:
Alright, I removed the wildcard imports, I think.